src/firstvoices/backend/tests/test_dictionary/test_parts_of_speech_view.py

```diff
@@ -0,0 +1,114 @@
+import json
+
+import pytest
+from django.core.management import call_command
+from rest_framework.reverse import reverse
+from rest_framework.test import APIClient
+
+from firstvoices.backend.models.constants import AppRole
+from firstvoices.backend.tests.factories import get_app_admin
+
+
+class TestPartsOfSpeechView:
+    """Tests for parts-of-speech views."""
+
+    FIXTURE_FILE = "partsOfSpeech_initial.json"
+    API_LIST_VIEW = "api:parts-of-speech-list"
+    API_DETAIL_VIEW = "api:parts-of-speech-detail"
+    APP_NAME = "backend"
+
+    @pytest.fixture(scope="session")
+    def parts_of_speech_db_setup(self, django_db_setup, django_db_blocker):
+        with django_db_blocker.unblock():
+            call_command("loaddata", self.FIXTURE_FILE)
+
+    def test_base_case_list_view(self, db, parts_of_speech_db_setup):
```

Comment:
We could parameterize this test and others to check multiple user types instead of just the staff role. Ex: https://github.com/First-Peoples-Cultural-Council/fv-be/blob/d575ba2be3ed3f21a751f703b42e7c3b60ee7522/firstvoices/backend/tests/test_predicates/test_predicate_base.py#L121-L128